lua/telescope/previewers.lua

```diff
@@ -374,6 +374,33 @@ previewers.vimgrep = defaulter(function(opts)
   }
 end, {})
 
+previewers.ctags = defaulter(function(opts)
```

Comment:
have you seen `previewers.help`? It might give you some ideas based on what `scode` looks like.

Reply:
Thanks for the insight :)